imports/plugins/included/social/client/components/pinterest.js

```diff
@@ -1,7 +1,7 @@
 import React, { Component } from "react";
 import PropTypes from "prop-types";
 import classnames from "classnames";
-import { Translation } from "/imports/plugins/core/ui/client/components";
+import { Components } from "/imports/plugins/core/components/lib";
```

Comment:
Should we import instead from `import { Components } from "@reactioncommerce/reaction-components";`?